src/app/api/v1/slack/confirmation/route.ts

```diff
@@ -50,11 +50,13 @@ export async function GET(
     // Save the code and the encription
     await new PostgresClient('', '')
       .putItems<Organization>([organization], TableName.Organizations);
-    return NextResponse.redirect(req.nextUrl.host + '/dashboard');
+
+    // Construct the full URL for redirection
+    const redirectUrl = new URL('/dashboard', `https://${req.headers.get('host')}`);
+    return NextResponse.redirect(redirectUrl.toString());
```

Comment:
redirects to the real dashboard, however we need to enhance the dashboard if no rotation exists. maybe show them a different page